src/Symfony/Bundle/WebServerBundle/WebServer.php

```diff
@@ -166,6 +173,8 @@ private function createServerProcess(WebServerConfig $config)
 
     private function getDefaultPidFile()
     {
-        return getcwd().'/.web-server-pid';
+        return $this->projectDir
```

Comment:
`return ($this->projectDir ?? getcwd()).'/.web-server-pid';`?

Reply:
Done